packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -169,6 +352,90 @@ class StatusColumn extends ColumnData<LinkData>
+    Widget Function() defaultHeaderRenderer,
+  ) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        const Text('Status'),
+        PopupMenuButton<StatusFilterOption>(
+          onSelected: (option) {
+            controller.updateFilterOptions(statusOption: option);
+          },
+          itemBuilder: (BuildContext context) {
+            var statusFilterOptions = <StatusFilterOption>[];
+            switch (tableViewType) {
+              case TableViewType.singleUrlView:
+                statusFilterOptions = [
+                  StatusFilterOption.failedAllCheck,
+                  StatusFilterOption.failedDomainCheck,
+                  StatusFilterOption.failedPathCheck,
+                  StatusFilterOption.noIssue,
+                ];
+                break;
+              case TableViewType.domainView:
+                statusFilterOptions = [
+                  StatusFilterOption.failedDomainCheck,
+                  StatusFilterOption.noIssue,
+                ];
+                break;
+              case TableViewType.pathView:
+                statusFilterOptions = [
+                  StatusFilterOption.failedPathCheck,
+                  StatusFilterOption.noIssue,
+                ];
+                break;
+            }
+            return statusFilterOptions
+                .map(
+                  (value) => PopupMenuItem<StatusFilterOption>(
+                    value: value,
+                    child: Row(
+                      children: [
+                        ValueListenableBuilder<StatusFilterOption>(
+                          valueListenable:
+                              controller.statusFilterOptionNotifier,
+                          builder: (context, option, _) => Checkbox(
+                            value: option == value,
+                            onChanged: (bool? checked) {
+                              if (checked!) {
+                                controller.updateFilterOptions(
+                                  statusOption: value,
+                                );
+                              } else {
+                                controller.updateFilterOptions(
+                                  statusOption: StatusFilterOption.showAll,
+                                );
+                              }
+                            },
+                          ),
+                        ),
+                        if (value == StatusFilterOption.failedAllCheck)
+                          const Text('Failed domain and path checks'),
+                        if (value == StatusFilterOption.failedDomainCheck)
+                          const Text('Failed domain checks'),
+                        if (value == StatusFilterOption.failedPathCheck)
+                          const Text('Failed path checks'),
+                        if (value == StatusFilterOption.noIssue)
+                          const Text('No issues found'),
```

Comment:
this can be simplified if you add the description to the StatusFilterOption values:
```
enum StatusFilterOption
  failedAllCheck('Failed domain and path checks'),
  ...;

  StatusFilterOption(this.description);

  final String description;
}
```
then here you can just write `value.description,`